tests/std/tests/P1614R2_spaceship/test.cpp

```diff
@@ -460,6 +462,36 @@ void ordering_test_cases() {
         static_assert(std::is_same_v<std::char_traits<char32_t>::comparison_category, std::strong_ordering>);
         static_assert(std::is_same_v<std::char_traits<wchar_t>::comparison_category, std::strong_ordering>);
     }
+    { // charconv
+        char c[7] = "123456";
+        int d;
+        std::from_chars_result a1 = std::from_chars(c, c + 6, d);
+        std::from_chars_result a2 = std::from_chars(c, c + 6, d);
+        std::from_chars_result a3 = std::from_chars(c + 1, c + 5, d);
+        std::from_chars_result a4 = std::from_chars(c - 10, c + 10, d);
+
+        assert(a1 == a2);
+        assert(a1 != a3);
+        assert(a1 != a4);
+
+        std::to_chars_result b1 = std::to_chars(c, c + 6, 123456);
+        std::to_chars_result b2 = std::to_chars(c, c + 6, 123456);
+        std::to_chars_result b3 = std::to_chars(c + 1, c + 5, 2345);
+        std::to_chars_result b4 = std::to_chars(c, c + 6, NAN);

```

Comment:
There are a couple of issues with these tests. First, `from_chars(c - 10, c + 10, d)` is reading outside of the `c` buffer, which is undefined behavior. Second, all of the `to_chars` calls will succeed, so there's no test coverage of the `errc` component.

Because these are `struct`s, it would be simpler and more direct to directly form the results - no need to call `from_chars`/`to_chars`, and we're already including `<system_error>` so we can mention `errc`.